app/Jobs.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Jobs extends Model
```

Comment:
Model name should always be singular. As an object of this class will represent a single job.
```suggestion
class Job extends Model
```